docs/book/features/integrations.md

```diff
@@ -49,12 +49,12 @@ These are the third-party integrations that ZenML currently supports:
 | pandas                    | ✅      | Exploration            |                                                                                               |                                                                                                                                                          |
 | Plotly                    | ✅      | Visualizer             | For Pipeline and PipelineRun visualization objects.                                           | [lineage](https://github.com/zenml-io/zenml/tree/main/examples/lineage)                                                                                  |
 | PyTorch                   | ✅      | Training               |                                                                                               |                                                                                                                                                          |
-| PyTorch Lightning         | ✅      | Training               |                                                                                               |                                                                                                                                                          |
+| PyTorch Lightning         | ✅      | Training               |                                                                                               | [pytorch](https://github.com/zenml-io/zenml/tree/main/examples/pytorch)                                                                                  |
```

Comment:
I think this ended up in the wrong line, should be with `pytorch` instead right?